gamefixes-egs/umu-1361510.py

```diff
@@ -0,0 +1,2 @@
+../gamefixes-steam/1361510.py
```

Comment:
This file needs to be a symbolic link. Within `gamefixes-egs`, run:

`ln -s ../gamefixes-steam/1361510.py umu-1361510.py`